pyspecProcScripts/phasing.py

```diff
@@ -336,7 +337,7 @@ def real_imag_mirror(forplot, ax):
         real_imag_mirror(forplot, ax2)
         ax2.set_title('shift=0')
         #}}}
-        fl.next('cost functions')
+        fl.next('Hermitian Diagnostic - Cost Functions')
```

Comment:
so, this plot is the one with the xlim problem that we discuss in the other comments.  This is the only real problem with the code that I see offhand.

Reply:
okay and getting rid of the xlims fixes the issue :)